src/i18n/it/README.md

```diff
@@ -0,0 +1,56 @@
+# 📖 Glossario
+
+Il glossario è una raccolta di vocaboli, modi di dire e pratiche comuni per la traduzione italiana della docmentazione di Astro.
+
+Alcune parole legate ad Astro e allo sviluppo web non vanno tradotte in quanto sono parte integrante del suo ambiante e non hanno una traduzione immediata nell'Italiano.
+
+## 🔄️ Vocaboli che non hanno bisogno di traduzione
+
+> 💡 Queste parole sono considerate nomi propri data la loro origine Inglese :
+
+| Vocabolo          | Definizione | Uso         |
+|-------------------|-------------|-------------|
+| Runtime           | Ambiente d'esecuzione / Tempo di esecuzione | Usato come "runtime di JavaScript" o "runtime edge" |
+| Framework         | Parola comunemente usata nel codice per specificare un software o un pacchetto che fa uso di una sintassi specifica al suo ambiente (es: `React`, `Vue`, `Svelte`, ecc...) | Utilizzato in riferimento a vari ambienti che Astro integra e supporta
+| Build             | Nel contesto di un lingaggio di programmazione che richiede un passaggio di compilazione e/o traduzione (tra due diversi linguaggi) | Questa parola deve essere usata quando si parla della fase di compilazione tramite il comando `astro build` usato per produrre un risultato compatibile con i browser e ottimizzato da Astro
+| Frontend          | Tutto quell'insieme di pacchetti e programmazione usati per costruire l'interfaccia utente direttamente visibile nel browser | Comunemente usato per distinguere il codice relativo alla logica dell'interfaccia utente dal codice che viene eseguito sul server e non è visibile all'utente |
+| Backend           | L'insieme del codice che veine usato per definire la struttura e il funzionamento del server | Vocabolo usato in contrapposizione a "Frontend" per distinguere codice e logica scritti per il browser da quelli scritti per il server |
+| Markdown, Astro, JavaScript, TypeScript, React, Vue, Svelte, Lit, Solid | Nomi di pacchetti o linguaggi di programmazione sempre scritti con la maiuscola | Utilizzati ovunque nella documentazione e sempre scritti con la maiuscola |
+| API (Application Programming Interface) | Interfaccia di programmazione dell'applicazione ovvero l'insieme dei metodi, classi e funzioni che definiscono il funzionamento e l'uso di una libreria o un framework | Abbreviazione usata in riferimento all'architettura e interfaccia di programmazione di Astro e altri software che potrebbero essere integrati con il framework |
+
+## 📚 Traduzioni comuni
+
+Alcuni vocaboli hanno una correlazione diretta o quasi con l'Italiano e vanno usate sempre allo stesso modo per mantenere una certa coerenza all'interno della documentazione:
+
+| Inglese                      | Italiano                                           |
+|------------------------------|----------------------------------------------------|
+| SSR / Server-side rendering  | SSR / Rendering lato server                        |
```

Comment:
I like "rendering" much more than "renderizzazione", I just want to highlight that we already have a few "renderizzazione" in our translations. Maybe we should review and possibly updated this term later.

Reply:
Yeah, "renderizzazione" is a mouthful... Even though it exists and would be the correct translation I see the english term being used way more and not only in the dev space but also in videography and digital art with tools like Premiere Pro and Photoshop.

Reply:
👍  for "rendering". What @milovangudelj said is correct; the term rendering is used on other spaces other than the web; for example, game engines too.